app/src/main/res/values/strings.xml

```diff
@@ -497,6 +497,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="enable_quest_confirmation_title">Enable this quest type?</string>
     <string name="default_disabled_msg_go_inside">This quest type is disabled by default because you will likely have to enter the place to be able to answer a quest of this type.</string>
     <string name="default_disabled_msg_go_inside_regional_warning">This quest type is disabled by default because it is not useful in some regions and you will likely have to enter the place to be able to answer a quest of this type.</string>
+    <string name="default_disabled_msg_tricky_3d_rendering">This quest type is disabled by default because it is time-consuming, correct answer is not always obvious and data is used only in 3D rendering.</string>
```

Comment:
How about this? I find it more important to mention that it would be more efficient in most cases to record this information from aerial imagery (=not with this app) than what the usages might be.
```suggestion
    <string name="default_disabled_msg_tricky_3d_rendering">This quest type is disabled by default because the roof shapes are often not well visible from the street. This quest type is also quite time-consuming, while it is in most cases easier and more efficient to map this from aerial imagery at home.</string>
```

Reply:
It misses the part that convinced me that it is a good idea (roof shapes not really matching to any selection), but maybe it is not needed to mention it...